### PR TITLE
Feat/update color scheme

### PR DESCRIPTION
**Pull request by Z-AOAO**
## 📝 Description

Add a brief description.

## 💣 Is this a breaking change (Yes/No):

- [ ] Yes
- [x] No

## 🚧 How to migrate?

Don't need.

## 📝 Additional Information

lol

### CODE REVIEW

File: packages/link/src/style.tsx [hunk 1]
```diff
@@ -14,6 +14,8 @@ export const colorSchemes: LinkColorScheme[] = [
   "blue",
   "cyan",
   "purple",
+  "tech-pink",
```
Comment (smallSohoSolo) [inline packages/link/src/style.tsx:17]: 驼峰

File: packages/progress/src/common-style.tsx [hunk 2]
```diff
@@ -15,6 +15,8 @@ const colorSchemes: ProgressColorScheme[] = [
   "blue",
   "cyan",
   "purple",
+  "tech-pink",
```
Comment (smallSohoSolo) [inline packages/progress/src/common-style.tsx:18]: 驼峰

File: packages/select/src/style.tsx [hunk 3]
```diff
@@ -14,6 +14,9 @@ const innerColor = [
   "orange",
   "cyan",
   "white",
+  "grayBlue",
+  "tech-pink",
```
Comment (smallSohoSolo) [inline packages/select/src/style.tsx:18]: 驼峰

File: packages/switch/src/style.tsx [hunk 4]
```diff
@@ -14,8 +14,8 @@ const innerColor = [
   "cyan",
   "purple",
   "grayBlue",
-  "brand-purple",
-  "brand-pink",
+  "tech-purple",
```
Comment (smallSohoSolo) [inline packages/switch/src/style.tsx:17]: 驼峰

File: packages/tag/src/style.tsx [hunk 5]
```diff
@@ -126,10 +126,12 @@ export const colors: TagColorScheme[] = [
   "blue",
   "cyan",
   "purple",
+  "tech-purple",
```
Comment (smallSohoSolo) [inline packages/tag/src/style.tsx:129]: 驼峰

File: packages/trigger/src/style.tsx [hunk 6]
```diff
@@ -17,6 +17,8 @@ const colorSchemes = [
   "blue",
   "cyan",
   "purple",
+  "tech-pink",
```
Comment (smallSohoSolo) [inline packages/trigger/src/style.tsx:20]: 驼峰

**Review by xiaoyu-a — APPROVED** [review]

**Review by smallSohoSolo — APPROVED** [review]

**Pull request merged**